src/findex.rs

```diff
@@ -265,4 +261,49 @@ mod tests {
             res
         );
     }
+
+    #[cfg(feature = "redis-store")]
+    fn get_redis_url() -> String {
+        if let Ok(var_env) = std::env::var("REDIS_HOST") {
+            format!("redis://{var_env}:6379")
+        } else {
+            "redis://localhost:6379".to_string()
+        }
+    }
+
+    #[tokio::test]
+    #[cfg(feature = "redis-store")]
+    async fn test_redis_insert_search_delete_search() {
```

Comment:
If correctly implemented, changing the memory should not influence the correctness of Findex. I agree though that this is in theory and testing is better than trusting. However, this test re-implements the exact same logic as the previous one : why not making the test generic instead?

Moreover, this is `findex.rs`, I would rather like not seeing any memory related code there. Move this test to `src/tests/findex-redis.rs`.